src/config/appDataTypes.ts

```diff
@@ -8,7 +8,7 @@ enum APP_DATA_TYPES {
 }
 
 enum APP_DATA_VISIBILITY {
-  MEMBER = 'member',
+  MEMBER = 'account',
```

Comment:
You should revert that, this did not change.
Or maybe use the one in sdk.

Reply:
Good catch ! Thankfully it was not used in the app, I think it was legacy from before we added the `AppDataVisibility` in sdk.